File: Bubble_Sort/Bubble_Sort.py
def swap(arr,i,j):
	temp = arr[i]
	arr[i] = arr[j]
	arr[j] = temp
	return

def bubbleSort(arr):
	isSorted = False
	lastUnsorted = len(arr)-1
	while not isSorted:
		isSorted = True
		for i in range(0,lastUnsorted):
			if arr[i] > arr[i+1]:
				swap(arr,i,i+1)
				isSorted = False
		lastUnsorted = lastUnsorted -1
	return

n = int(input())
arr = []

for i in range(0,n):
	arr.append(int(input()))

bubbleSort(arr)
print(arr)


		

File: Bubble_Sort/Bubble_Sort.java
import java.util.Scanner;

public class Bubble_Sort{

	public static void swap(int[] arr, int i, int j){
		int temp = arr[i];
		arr[i] = arr[j];
		arr[j] = temp;
	}

	public static void bubbleSort(int[] arr){

		boolean isSorted = false;
		int lastUnsorted = arr.length-1;
		while(!isSorted){
			isSorted = true;
			for (int i=0;i<lastUnsorted;i++ ) {
				if(arr[i] > arr[i+1]){
					swap(arr,i,i+1);
					isSorted = false;
				}
			}
			lastUnsorted--;
		}

		return;

	}

	public static void printArray(int[] arr){
		String output = "[";
		for (int i=0;i<arr.length;i++) {
			output += " "+arr[i]+" ";
			if(i<arr.length-1){
				output +=",";
			}
		}
		output +="]";
		System.out.println(output);
		return;
	}

	public static void main(String[] args) {
		
		Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		int[] arr = new int[n];

		for (int i=0;i<n ;i++ ) {
			arr[i] = input.nextInt();
		}
		bubbleSort(arr);
		printArray(arr);
	}
}